src/panels/config/entities/ha-config-entities.ts

```diff
@@ -322,6 +353,15 @@ export class HaConfigEntities extends SubscribeMixin(LitElement) {
                   "ui.panel.config.entities.picker.filter.show_unavailable"
                 )}
               </paper-icon-item>
+              <paper-icon-item @tap="${this._showReadOnlyChanged}">
```

Comment:
I don't know if this should be an option. They are active, they have states etc otherwise they wouldn't exist. I think that we should always show them.